docs/development/developer-setup.md

```diff
@@ -71,6 +71,35 @@ docker compose up
 bin/setup
 ```
 
+## Using sanitised production data
+
+You may occasionally want to use data as close to production data ase possible, for instance during an incident or when preparing for a large migration.
+
+### 1. Downloading the sanitised data
+
+- Login to azure with your `digitalauth.education.gov.uk` address and request a PIM (speak to infra or a fellow developer for instructions on how to do this)
+- Once the PIM is approved, you can download the file via the AZ interface by navigating Storage accounts -> `s189p01attdbbkppdsa` -> Containers -> database-backup -> `att_backup_sanitised.sql.gz`
+- Unzip the file, delete the zip file
+
+### 2. Replacing your database
+
+- run `bin/rails db:drop` and `bin/rails db:create`; do not run migrations
+- run `psql bat_apply_development < ~/Downloads/att_backup_sanitised.sql` (or wherever your downloads end up)
+- make sure you delete the sql file
+
+### 3. Creating a dev-support user
+The production dump won't have a support user for bypassing DFE login, so you'll need to create one before you can login to the application.
+
+```ruby
+
+SupportUser.create!(
+  dfe_sign_in_uid: 'dev-support',
+  email_address: 'support@example.com',
+  first_name: 'Susan',
+  last_name: 'Upport'
```

Comment:
I wonder why they used the last_name `Upport` and not `Support` back in the day 🤔

Reply:
A mystery!